.github/workflows/quality.yml

```diff
@@ -0,0 +1,34 @@
+name: Quality
+
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+    branches:
+      - dev
+
+jobs: 
+  eslint: 
+    # Executes `lint:fix` to clean up errors or 
+    # warnings that are autofixable.
+    # Also posts warnings or errors that are not autofixable.
+    name: Eslint Fix
+    runs-on: ubuntu-latest
+    permissions: 
+      contents: write
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+
+      - name: Install deps
+        run: yarn
+      
+      - name: Run ESLint Fix
+        run: yarn lint:fix
+
+      - uses: stefanzweifel/git-auto-commit-action@v5
```

Comment:
See the [auto commit repo README on use with forks from public repo](https://github.com/stefanzweifel/git-auto-commit-action/tree/v5/#use-in-forks-from-public-repositories) as this would explain the failing check here. It also says that the suggested approach has a security warning to it and that running it only on push to the main branch (`dev` in our case) would be the best.

Would like thoughts here, especially considering running `lint:fix` as an action.